schemas/authentication-credentials/authentication-credentials.json

```diff
@@ -0,0 +1,48 @@
+{
+  "$id": "https://raw.githubusercontent.com/cyentific-rni/cacao-json-schemas/cacao-v2.0-csd01/schemas/authentication-credentials/authentication-credentials.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "authentication-credentials",
+  "description": "In a CACAO playbook, authentication credentials are used by agents and targets when they need to authenticate against a resource. Authentication credentials are stored in a dictionary where the ID is the key and the value is an 'authentication-credentials' object (see section 10.2). Common properties for authentication credentials are defined in section 6.1. Authentication credentials can use and refer to variables just like other parts of the playbook. For any authentication method property value, the producer may define a variable substitution such that the actual property value is determined at runtime based on the variable assigned to the credential. In Example 6.1, an authentication method is referenced within an agent, but the authentication method's actual values are based on variables instead of being hard-coded in the credential itself. It is generally best practice to always use variables for authentication credentials. Each authentication object contains some base properties that are common across all objects. These properties are defined in the following table. The ID for each object is stored as the key in the 'authentication_credentials_definitions' dictionary.",
+  "type": "object",
+  "properties": {
+    "type": {
+      "$ref": "#/$defs/authentication-credentials-type-ov",
+      "description": "The type of object being used. The value of this property SHOULD come from the 'authentication-credentials-type-ov' vocabulary."
+    },
+    "name": {
+      "type": "string",
+      "description": "The name that represents this object that is meant to be displayed in a user interface or captured in a log message. This property SHOULD be populated"
+    },
+    "description": {
+      "type": "string",
+      "description": "More details, context, and possibly an explanation about this object."
+    },
+    "authentication_credentials_extensions": {
+      "type": "object",
```

Comment:
"$ref": "../data-types/dictionary.json" ?

Reply:
Mistake, fixed in the next commit.